deps/leader_cron-0.1/.gitignore

```diff
@@ -0,0 +1,11 @@
+erl_crash.dump
```

Comment:
@siplabs This is a general comment on leader_cron so I am just adding it to the first of the files.  This is using native distributed Erlang which is NOT currently part of our platform.  I am opposed to such a significant and fundamental change as to start requiring interconnected Erlang nodes for this.  I have looked at gen_leader before and am of the opinion that it would be very feasible to use AMQP (gen_listener) as the message passing.  In some places it simplifies it greatly because rather than track there is a the ability to broadcast.

So here is the dilemma if we accept this it will become messy as those that wish to use it interconnect their whapps nodes (bypassing AMQP and effectively creating two message buses).  I also have concerns that the scope of this whapps interconnect would creep into other things.  As such I would like to start discussions on how to achieve this same functionality using the existing message bus before we accept the PR.